scripts/install-binary.js

```diff
@@ -6,18 +6,28 @@ const BinBuild = require('bin-build')
 const path = require('path')
 const tempfile = require('tempfile')
 const fs = require('fs')
+const download = require('download')
+
+const platform = process.platform
+const arch = process.arch
 
 const JQ_INFO = {
   name: 'jq',
   url: 'https://github.com/stedolan/jq/releases/download/',
   version: 'jq-1.5'
 }
 
+const JQ_NAME_MAP = {
+  'def': 'jq',
```

Comment:
`default` is kind of reserved, then i must change the `JQ_NAME_MAP.def` to `JQ_NAME_MAP.['default']`...
How important is this for you?